packages/amplify-category-function/src/commands/function/build.ts

```diff
@@ -1,18 +1,43 @@
-import { category as categoryName } from '../../constants';
+import { $TSContext } from 'amplify-cli-core';
+import { ServiceName } from '../..';
+import { category } from '../../constants';
+import { PackageRequestMeta } from '../../provider-utils/awscloudformation/types/packaging-types';
+import { buildFunction } from '../../provider-utils/awscloudformation/utils/buildFunction';
+import { packageResource } from '../../provider-utils/awscloudformation/utils/package';
 
-const subcommand = 'build';
+export const name = 'build';
 
-module.exports = {
-  name: subcommand,
-  run: async context => {
-    const { amplify, parameters } = context;
-    const resourceName = parameters.first;
+/**
+ * To maintain existing behavior, this function builds and then packages lambda functions
+ */
+export const run = async (context: $TSContext) => {
+  const resourceName = context?.input?.subCommands?.[0];
+  const cont =
```

Comment:
nit: is this supposed to be short for 'continue'? If so, I suggest spelling it out

Reply:
`continue` is a reserved word. Renaming to `confirmContinue`